### PR TITLE
[MPQEditor] Introduce MPQEditor

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit introduces MPQEditor for manual mixed precision quantization task.

Fresh draft: https://github.com/Samsung/ONE-vscode/pull/1511
Previous draft: https://github.com/Samsung/ONE-vscode/pull/1505
Related: https://github.com/Samsung/ONE-vscode/issues/1491

ONE-vscode-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

File: src/MPQEditor/MPQEditor.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  }
+
+  /**
+   * Get the static html used for the editor webviews.
+   */
+  private async initWebview(
+    document: vscode.TextDocument,
+    webviewPanel: vscode.WebviewPanel
+  ): Promise<void> {
+    let webview: vscode.Webview = webviewPanel.webview;
```
Comment (dayo09) [inline src/MPQEditor/MPQEditor.ts:72]:
```suggestion
    const webview: vscode.Webview = webviewPanel.webview;
```

(optional)

**Review by dayo09 — DISMISSED** [review]
LGTM :-D

**Review by dayo09 — APPROVED** [review]
I see that you have added the basic features, LGTM.

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**